src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -1020,6 +1021,52 @@ export const handleUpdateSettings: RequestHandler<
     })
 }
 
+/**
+ * Handler for GET /form/:formId/settings.
+ * @security session
+ *
+ * @returns 200 with latest form settings on successful update
+ * @returns 401 when current user is not logged in
+ * @returns 403 when current user does not have permissions to obtain form settings
+ * @returns 404 when form to retrieve settings for cannot be found
+ * @returns 409 when saving form settings incurs a conflict in the database
+ * @returns 500 when database error occurs
+ */
+export const handleGetSettings: RequestHandler<
+  { formId: string },
+  FormSettings | ErrorDto
+> = (req, res) => {
+  const { formId } = req.params
+  const sessionUserId = (req.session as Express.AuthedSession).user._id
+
+  return UserService.getPopulatedUserById(sessionUserId)
+    .andThen((user) =>
+      // Retrieve form for settings as well as for permissions checking
+      FormService.retrieveFullFormById(formId).map((form) => ({
+        form,
+        user,
+      })),
+    )
+    .andThen(AuthService.checkFormForPermissions(PermissionLevel.Read))
+    .map((form) =>
+      res.status(StatusCodes.OK).json(form.getSettings() as IPopulatedForm),
```

Comment:
why is this type cast necessary? I tried removing it and it seems to work fine, plus we've specified in the `RequestHandler` generic args that we are returning a `FormSettings`